cmd/bbolt/command_check_test.go

```diff
@@ -0,0 +1,33 @@
+package main_test
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	main "go.etcd.io/bbolt/cmd/bbolt"
+	"go.etcd.io/bbolt/internal/btesting"
+)
+
+func TestCheckCommand_Run(t *testing.T) {
```

Comment:
while we're at it, can you add some test that exercises the corruption?

Reply:
This PR just migrates the check command to cobra style. This comment is an enhancement request. It's OK to discuss it separately. 

I think the purpose of CLI's test cases are just to verify the CLI itself instead of the functionalities. Also we already some test cases to verify the corrupted pages. FYI. https://github.com/etcd-io/bbolt/blob/main/tx_check_test.go

Reply:
yes the corruption check has been test already, this test case just to verify the cli is working as expected 👍🏽

Reply:
well, part of testing that CLI is also ensuring this case works. Feel free to do it as a follow-up PR.

Reply:
i am confused now, so is this 

> This comment is an enhancement request

or just adding new test for the corruption check, because we already have tests for this https://github.com/etcd-io/bbolt/blob/main/tx_check_test.go

just to make sure we are all on the same page